app/config/Config.scala

```diff
@@ -63,6 +63,7 @@ object Config extends AwsInstanceTags {
 
   lazy val sharedSecret: String = config.getConfigStringOrFail("api.sharedsecret")
 
+  lazy val incopyOpenUrl: String = "gnm://openinincopy/${storyBundleId}/checkout/readwrite"
```

Comment:
This is incorrect, it should read:
`gnm://openinincopy/storybundle/${storyBundleId}/checkout/readwrite`
cc @hoyla